yarf/rf_libraries/libraries/ocr/rapidocr.py

```diff
@@ -80,8 +83,8 @@ def find(
         self,
         image: Image.Image | Path,
         text: str,
-        confidence: float = DEFAULT_CONFIDENCE,
-        coincidence: float = DEFAULT_COINCIDENCE,
+        confidence: float | None = None,
+        coincidence: float | None = None,
```

Comment:
Can you explain this change?

Reply:
Using 

```
        confidence: float = DEFAULT_CONFIDENCE,
        coincidence: float = DEFAULT_COINCIDENCE,
```

has the effect that the default values of the arguments are always what `DEFAULT_CONFIDENCE` and `DEFAULT_COINCIDENCE` evaluated to at the time the function was defined (i.e. when the module is first imported, see  https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

The change makes it evaluate `DEFAULT_CONFIDENCE` and `DEFAULT_COINCIDENCE` at the time the function is called.